litmus-portal/frontend/src/components/Unimodal/styles.ts

```diff
@@ -0,0 +1,58 @@
+import { makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles((theme) => ({
+  modalContainer: {
+    width: '70%',
```

Comment:
Also fix the height to 80 or 70% what seems best to you and make the content inside vertically scrollable. That would abstract out most of the reusability of the modal.
```
overflow-x: hidden;
overflow-y: auto;
```